app/service/service_senders_schema.py

```diff
@@ -0,0 +1,11 @@
+add_service_email_reply_to_request = {
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "description": "POST service email reply to address",
+    "type": "object",
+    "title": "Add new email reply to address for service",
+    "properties": {
+        "email_address": {"type": "string", "format": "email_address"},
+        "is_default": {"type": "boolean"}
+    },
+    "required": ["email_address"]
```

Comment:
would it make it easier if we demand that is_default is always set? i'm a fan of more restrictive inbound schemas in general. Not super fussed tho.

Reply:
It is more straight forward